config/environments/production.rb

```diff
@@ -1,17 +1,18 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
-  config.action_mailer.default_url_options = {host: ENV["DEFAULT_URL_HOST"]} # for devise authentication
+  config.action_mailer.default_url_options = { host: ENV["DOMAIN"] }
   config.action_mailer.raise_delivery_errors = true
-  config.action_mailer.perform_deliveries = true
+  # Do not send emails in staging or qa
+  config.action_mailer.perform_deliveries = ENV["APP_ENVIRONMENT"] == "production"
   config.action_mailer.delivery_method = :smtp
   config.action_mailer.smtp_settings = { # WARNING do not let standardrb linter change this block, it breaks
-    :address => 'smtp-relay.sendinblue.com',
-    :port => 587,
-    :user_name => ENV["SENDINBLUE_EMAIL"],
-    :password => ENV["SENDINBLUE_PASSWORD"],
-    :authentication => 'login',
-    :enable_starttls_auto => true
+    address: 'smtp-relay.sendinblue.com',
```

Comment:
In the past, this change has made email stop working. See the line above `# WARNING do not let standardrb linter change this block, it breaks` This seems unreasonable but it has been true in the past and it is the reason why the linter explicitly ignores this file. Hopefully this was an artifact of some other weirdness and is not actually true. Please remove the comment also :)
I will merge this and test it carefully 😓

Reply:
Oh, that seems very strange. I guess let's see how it goes.